### PR TITLE
O3-2895: Fix Layout of Patient Search Patient Banner

### PR DESCRIPTION
**Pull request by mogoodrich**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

File: packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.scss [hunk 1]
```diff
@@ -90,6 +90,7 @@
   @include type.type-style('body-compact-02');
   color: $text-02;
   margin-top: 0.375rem;
+  text-align: left;
```
Comment (mogoodrich) [inline packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.scss:93]: The demographics and the identifiers were bought center aligned (see screen shots) this and the line below just left-aligns them.

File: packages/esm-patient-search-app/src/ui-components/overlay/overlay.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
   top: spacing.$spacing-09;
   right: 0;
   height: 100vh;
-  min-width: 33rem;
+  width: 26.25rem;
```
Comment (mogoodrich) [inline packages/esm-patient-search-app/src/ui-components/overlay/overlay.scss:11]: Changing the overlay to the standard 420px workspace/overlay size

**Comment by denniskigen** [comment]
@mogoodrich Could we indent the overlay heading by 1rem on either side? It'd look a lot better if it weren't flush with the left border of the overlay.

**Comment by mogoodrich** [comment]
> @mogoodrich Could we indent the overlay heading with a horizontal margin of 1rem? It'd look a lot better if it weren't flush with the left border of the overlay. Also, the Close button on the overlay should be flush with the right border.

Yep, @denniskigen ... the overlay layout is somewhat screwed up.  The coloring of the button is wrong, too, as you can tell.  I started looking into that this morning but wasn't immediately able to track down how to fix and wanted to get this PR out this week... I can either fix in a follow-on commit or ticket.

**Review by mseaton — APPROVED** [review]
LGTM

**Comment by mogoodrich** [comment]
@denniskigen  are you okay with the PR if the overlay heading is ticketed separately?  @ibacher  @brandones  do you want to add any thoughts before I merge?

**Comment by mogoodrich** [comment]
Going to merge this in because I'm running into some weird layout issues with another commit and need to see if this fixes it.  Post commit reviews still welcome.

**Pull request merged**